### ISSUE TITLE
Target helper missing on saves

### ISSUE THREAD

**Issue by spookymonster** [reonZ/pf2e-toolbelt#32]
PF2E Target Damage shows an 'add target' icon in the spell saves button, but PF2E Toolbelt doesn't. Instead, it only shows the 'add target' button on the damage rolls.
![image](https://github.com/reonZ/pf2e-toolbelt/assets/16195156/3d412059-bc24-4dc9-ae90-d713b1291d45)

**Comment by spookymonster** [comment]
A shame. Toolbelt's implementation is too clumsy for regular use (select tokens, click Roll Save button, click Roll Damage, set selected tokens as targets, click Add Targets, apply damage). And because it doesn't add the targets before the save is rolled, it doesn't highlight the damage buttons (like Target Damage does).

I'll have to go re-install the last working version of Target Damage and lock it in until it breaks and/or this is added to Toolbelt.

Thanks for the quick response.

**Comment by reonZ** [comment]
none of that is true, target your tokens and click damage, everything is in the damage card, as explained in the readme...

**Comment by spookymonster** [comment]
Thanks for responding. I have confirmed that the 'running man' icon on the individual target damage row is the saving throw roll button. It's different from how the old mod worked, but I can see the advantages once you get used to it. Having the saves appear in the damage message instead of separate messages minimizes chat log clutter.

2 suggestions:
- Make the buttons look like buttons :). I didn't click on them because I thought they were informational icons ("The ability to roll saves can directly be handled from there as well" doesn't make their function clear, either)
- Consider giving the 'roll all saves' action a separate button, instead of the right-click context menu.

Again, thanks for taking the time to reply. I appreciate your time and effort on this project. I have updated my previous post accordingly.
